### ISSUE TITLE
Get lab RIM projects 

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/RIM#15]
@wally-mac, I know the lab has done a bunch of RIM mapping for projects (e.g. Bailey Flat Klamath, BLM Utah, USFS PIBO).Do you know if these were done using RIM and therefor exist as riverscapes projects? If so, we're uploading a bunch for the USFS/PIBO thing from @kbartelt to warehouse (see #11). It would be really easy for us to fix at the same time yours? If not, we might want to scope with @Jdgilby and @jtgilbert what it would take to get your layers into the system.  @nick4rivers will also be interested in this.

**Comment by nick4rivers** [comment]
Yes, seeing how you are approaching this would really be helpful whether they are Riverscapes compliant or not.

**Comment by Cashe93** [comment]
@wally-mac and @joewheaton, I'll compile a list of the mapping we've done using drone imagery. However, I can confirm that, for the vast majority of these, the RIM tool was **not** used. The mapping exists, just not as riverscapes projects. I'll post the list here momentarily.

**Comment by wally-mac** [comment]
@Cashe93 please run RIM for the PIBO site in Idaho. I think this is a good opportunity to make those projects compliant. What would it take to make the other projects Riverscape compliant?

**Comment by Cashe93** [comment]
@wally-mac, we would just have to run RIM with what we have already mapped. If everything runs smoothly it shouldn't take much. However, the RIM tool doesn't include a handful of the geomorphic units that we've been mapping. It only uses dam crests, inundation, thalwegs, valley bottom, and valley bottom centerline. The other features we've been mapping won't be included unless we manually add them, and even then the statistics won't reflect those features.  Is this something we want to do, or should we just keep what's currently included in RIM?

**Comment by wally-mac** [comment]
@Cashe93 I think for now just use the existing functionality of RIM.  I think in the future @nick4rivers will be working on including the Tier 1 unit mapping but I think it will be in a different tool than RIM.  @nick4rivers can you fact check me.? Thanks

**Comment by Cashe93** [comment]
Sounds good. I'll get started on this tomorrow.

**Comment by nick4rivers** [comment]
@wally-mac and @Cashe93 thanks for keeping us in the loop and providing some of the raw data you all are producing. Yes, the idea here is to develop a QGIS plugin for creation and metric calculation for RIM projects. This should really help with efficiency and consistency across the board. It should also allow projects to easily find a home within the Riverscapes Warehouse.

Another quick question is whether your group is currently running the .py scripts within this repo? Or are you mostly manually creating the necessary collection of feature classes and calculating summary metrics.

Thanks all, don't hold your breath on this tool. It's gonna be a bit but we will try to keep you all in the conversation and hopefully can create something that is useful and gets used.

**Comment by Cashe93** [comment]
@nick4rivers, most of these projects were done manually without running the .py scripts. However, the few times I've toyed around with the RIM tool, I was using the .py scripts that were updated by @tyler1218hatch which are [located here](https://github.com/tyler1218hatch/RIM/tree/toolbox-dev). As mentioned above, many of our projects have required mapping more than just valley bottom and inundation, so it would be nice to have a tool that incorporates all of those features if possible.

**Comment by nick4rivers** [comment]
@Cashe93 that's super helpful - i'll reach out when I actually get to implementing this stuff.

**Comment by jtgilbert** [comment]
@Cashe93 have you used the Arc toolbox from Tyler's fork, or run the scripts independently?

**Comment by Cashe93** [comment]
I'm using the toolbox.

**Comment by jtgilbert** [comment]
@Cashe93 Joe doesn't want us to spin our wheels trying make the toolbox deployable if it's not easily diagnosed. For the lab projects you can manually put them into project structure (the way you guys have been? which could include additional layers like you were discussing) and use [these project xmls](https://usu.app.box.com/folder/114745079723?s=iwywy40hu9jzbsmdlaaf9eeiu0ogwaxe) as a template for RS compliance so that the projects can open with RAVE.
